cmd/lotus-storage-miner/run.go

```diff
@@ -30,7 +30,11 @@ var runCmd = &cli.Command{
 	Flags: []cli.Flag{
 		&cli.StringFlag{
 			Name:  "api",
-			Value: "",
+			Value: "2345",
+		},
+		&cli.StringFlag{
+				Name: "apihost",
+				Value: "127.0.0.1",
```

Comment:
maybe we should just have the api flag accept a multiaddr? We can also have a fallback where if it parses as a number, use the current behavior